README.md

```diff
@@ -52,7 +52,7 @@ The auxiliary container is:
 
```

Comment:
There are are multiple aux containers, so line 51 should read "The auxiliary containers are:" (plural container, plural is).

Reply:
It's done.